templates/base.html

```diff
@@ -107,17 +107,24 @@ <h1><a href="{{ SITEURL }}">{{ SITETITLE }}</a></h1>
       {% if pages or LINKS %}
       <nav>
         <ul class="list">
+          {# Open links in new window depending on the LINKS_IN_NEW_TAB setting #}
+          {% macro link_target(link) -%}
+            {% if LINKS_IN_NEW_TAB not in ('no', 'none', false, 0, 'external') or (LINKS_IN_NEW_TAB == "external" and not link.startswith("/") and not link.startswith(SITEURL)) %}
+            target="_blank"
+            {% endif %}
+          {% endmacro %}
+
           {% if PAGES_SORT_ATTRIBUTE -%}
             {% set pages = pages|sort(attribute=PAGES_SORT_ATTRIBUTE) %}
           {%- endif %}
           {% if DISPLAY_PAGES_ON_MENU %}
           {% for page in pages %}
-          <li><a href="{{ SITEURL }}/{{ page.url }}{% if not DISABLE_URL_HASH %}#{{ page.slug }}{% endif %}">{{ page.title }}</a></li>
+            <li><a {{ link_target(SITEURL) }} href="{{ SITEURL }}/{{ page.url }}{% if not DISABLE_URL_HASH %}#{{ page.slug }}{% endif %}">{{ page.title }}</a></li>
```

Comment:
`page.url` doesn't start with a `/`, that's why I pass `SITEURL`, so `LINKS_IN_NEW_TAB == "external"` works correctly.